db/collection.go

```diff
@@ -1032,7 +1100,12 @@ func (c *collection) create(ctx context.Context, txn datastore.Txn, doc *client.
 		return err
 	}
 
-	return c.indexNewDoc(ctx, txn, doc)
+	err = c.indexNewDoc(ctx, txn, doc)
+	if err != nil {
+		return err
+	}
+
+	return c.registerDocCreation(ctx, doc.ID().String())
```

Comment:
todo: I think the name of this method should make it clear that it deals with ACP stuff.

Reply:
Please let me know if you have any suggestions.

Actually might delete this method completely.

- [ ] rename or delete `registerDocCreation` method

Reply:
- [x] Renamed in `PR(ACP): Rename Registering Document Functions`